packages/graphql/schemas/schema.graphql

```diff
@@ -722,6 +722,7 @@ enum ErrorTypeEnum {
   BAD_POLICY_WARNING_TOOLTIP
   BROWSER_NOT_FOUND_BY_NAME
   BROWSER_NOT_FOUND_BY_PATH
+  BROWSER_UNSUPPORTED_LAUNCH_OPTION
```

Comment:
This error got added in a different PR, but the schema wasn't committed, so I just added it here. We should probably not commit generated files anyway, I think we can probably `gitignore` this file in a future PR.

Reply:
Committing the schema I think allows us to work offline or if the dashboard happens to be down which are nice guards to have in place. I wonder if there's a bot that could check and auto-PR schema changes once a day or whatever?